src/phone_sms_terms.js

```diff
@@ -104,6 +105,7 @@ $('form.external-ak').on('submit', function() {
       $('#id_email_box').hide();
     }
 
+    $('#id_phone_type').val('mobile');
```

Comment:
This isn't really part of this commit, but noting that we could give this function a name and reference it by name to avoid the duplicate code.